### PR TITLE
Bmoric/add feature flag table

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
This is adding a a feature flag table that will be store the feature flags value instead of the environment variable. This is needed for https://github.com/airbytehq/airbyte/issues/10021

### CODE REVIEW

**Review by subodh1810 — COMMENTED** [review]
Dont we also want a persistence class for the table ?

File: airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt [hunk 1]
```diff
@@ -86,6 +86,15 @@ create table "public"."connection_operation"(
       "operation_id"
     )
 );
+create table "public"."feature_flag"(
+  "id" uuid not null,
+  "name" varchar(256) not null,
+  "value" varchar(2147483647) null,
```
Comment (jrhizor) [inline airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt:92]: Why's this set so high instead using the default infinite for varchar or limiting it to 256 or similar?

**Review by jrhizor — APPROVED** [review]
only one small comment

File: airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt (hunk 1 above)
Comment (benmoriceau) [reply airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt:92]: It looks like max Int to me. This file is an autogenerated one which got created when running the dumpSchema command, I defined it without a limit here: https://github.com/airbytehq/airbyte/pull/10014/files#diff-3a3d6d3d328fb108a672f34a61ecfb6335a08c9f9f5953e1bf95c1911aadf0d2R37

**Comment by benmoriceau** [comment]
> the

Thanks for pointing that out. I will generate it in the related project.

File: airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt (hunk 1 above)
Comment (jrhizor) [reply airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt:92]: Oh forgot this was autogenerated. nvm.

**Pull request closed without merging**